RadarSDK/RadarSegment.h

```diff
@@ -0,0 +1,29 @@
+//
+//  RadarSegment.h
+//  RadarSDK
+//
+//  Copyright © 2019 Radar Labs, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ Represents a user segment.
+ */
+@interface RadarSegment : NSObject
+
+/**
+ The description of the segment.
+ */
+@property (nonnull, copy, nonatomic, readonly) NSString *_description;
```

Comment:
nit: is this a private variable due to the `_`? I'd assume `description` is meant to be available to clients

Reply:
It conflicts with `description` property on `NSObject` (https://developer.apple.com/documentation/objectivec/nsobjectprotocol/1418746-description). If I could go back in time, I would call this field `name`. Alas...